src/Cedar/Server.c

```diff
@@ -2514,6 +2514,8 @@ void SiInitDefaultHubList(SERVER *s)
 	SiSetDefaultLogSetting(&g);
 	SetHubLogSetting(h, &g);
 
+/*
+	// Add test user(s)

```

Comment:
Just remove the whole snippet. It is a dead piece of code anyway.
If we needed this snippet in the future, it is still saved in git history

Reply:
Done.